### DISCUSSION TITLE
There is a translation problem on the website

### DISCUSSION THREAD

**Discussion by NihatFarz** [community/community#58237] [New to GitHub]
There is a translation problem on the website

When the translation is active, the site gives an error when the page passes.

**Comment by the-officialjosh** [comment]
What website are you referring to is it GitHub or something you are working on? If it is GitHub you could try reaching out to GitHub support

**Reply by NihatFarz** [reply]
yes GitHub

**Reply by the-officialjosh** [reply]
Why not try using an extension like Google Translate for your browser? That is what I use. If that works please mark this reply as the answer so GitHub registers it
